master branch is broken

Currently, master branch is broken, because of `xnav` version upgrade from `0.1.7` -> `0.1.9`:

```text
java.lang.IllegalStateException: org.apache.maven.plugin.MojoFailureException: 'LintMojo' execution failed

	at org.eolang.maven.Moja.execute(Moja.java:126)
	at org.eolang.maven.FakeMaven.execute(FakeMaven.java:271)
	at org.eolang.maven.FakeMaven.execute(FakeMaven.java:168)
	at org.eolang.maven.LintMojoTest.savesVerifiedResultsToCache(LintMojoTest.java:308)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
Caused by: org.apache.maven.plugin.MojoFailureException: 'LintMojo' execution failed
	at org.eolang.maven.SafeMojo.exitError(SafeMojo.java:627)
	at org.eolang.maven.SafeMojo.execute(SafeMojo.java:481)
	at org.eolang.maven.Moja.execute(Moja.java:122)
	... 6 more
Caused by: java.util.concurrent.ExecutionException: java.io.UncheckedIOException: java.io.IOException: java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: 'void com.github.lombrozo.xnav.Xnav.<init>(java.lang.String)'
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:205)
	at org.eolang.maven.SafeMojo.execWithTimeout(SafeMojo.java:576)
	at org.eolang.maven.SafeMojo.execute(SafeMojo.java:463)
	... 7 more
Caused by: java.io.UncheckedIOException: java.io.IOException: java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: 'void com.github.lombrozo.xnav.Xnav.<init>(java.lang.String)'
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:58)
	at org.cactoos.iterator.Mapped$1.next(Mapped.java:68)
	at org.cactoos.iterator.IteratorEnvelope.next(IteratorEnvelope.java:59)
	at org.cactoos.scalar.Folded.value(Folded.java:82)
	at org.cactoos.number.NumberOfScalars.lambda$new$1(NumberOfScalars.java:68)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.number.NumberOfScalars.intValue(NumberOfScalars.java:93)
	at org.cactoos.number.NumberEnvelope.intValue(NumberEnvelope.java:55)
	at org.eolang.maven.util.Threaded.total(Threaded.java:90)
	at org.eolang.maven.LintMojo.exec(LintMojo.java:100)
	at org.eolang.maven.SafeMojo.lambda$execWithTimeout$7(SafeMojo.java:573)
	at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
	at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
	at java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)
	at java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:113)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:147)
	at java.base/java.util.concurrent.Executors$DelegatedExecutorService.submit(Executors.java:790)
	at org.eolang.maven.SafeMojo.execWithTimeout(SafeMojo.java:571)
	at org.eolang.maven.SafeMojo.execute(SafeMojo.java:463)
	at org.eolang.maven.Moja.execute(Moja.java:122)
	at org.eolang.maven.FakeMaven.execute(FakeMaven.java:271)
	at org.eolang.maven.FakeMaven.execute(FakeMaven.java:168)
	at org.eolang.maven.LintMojoTest.savesVerifiedResultsToCache(LintMojoTest.java:308)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:767)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.SameThreadTimeoutInvocation.proceed(SameThreadTimeoutInvocation.java:45)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$8(TestMethodTestDescriptor.java:217)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:156)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:146)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:144)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:100)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:160)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:146)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:144)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:100)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:160)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:146)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:144)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:100)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:198)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:169)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:93)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:58)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:141)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:57)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:103)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:85)
	at org.junit.platform.launcher.core.DelegatingLauncher.execute(DelegatingLauncher.java:47)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:63)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)
Caused by: java.io.IOException: java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: 'void com.github.lombrozo.xnav.Xnav.<init>(java.lang.String)'
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:56)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:57)
	at org.cactoos.scalar.Checked.wrappedException(Checked.java:99)
	at org.cactoos.scalar.Checked.value(Checked.java:84)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	... 94 more
Caused by: java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: 'void com.github.lombrozo.xnav.Xnav.<init>(java.lang.String)'
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at org.cactoos.scalar.Mapped.lambda$new$0(Mapped.java:58)
	at org.cactoos.scalar.ScalarEnvelope.value(ScalarEnvelope.java:53)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	... 96 more
Caused by: java.lang.NoSuchMethodError: 'void com.github.lombrozo.xnav.Xnav.<init>(java.lang.String)'
	at org.eolang.lints.LtByXsl.<init>(LtByXsl.java:90)
	at org.eolang.lints.PkByXsl.lambda$all$0(PkByXsl.java:81)
	at org.cactoos.scalar.Mapped.lambda$new$0(Mapped.java:58)
	at org.cactoos.scalar.ScalarEnvelope.value(ScalarEnvelope.java:53)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.iterator.Mapped$1.next(Mapped.java:68)
	at org.cactoos.iterator.IteratorEnvelope.next(IteratorEnvelope.java:59)
	at java.base/java.lang.Iterable.forEach(Iterable.java:74)
	at org.cactoos.list.ListOf.<init>(ListOf.java:66)
	at org.cactoos.list.ListOf.<init>(ListOf.java:56)
	at org.cactoos.iterator.Shuffled.lambda$new$0(Shuffled.java:67)
	at org.cactoos.scalar.Sticky.lambda$new$0(Sticky.java:73)
	at org.cactoos.func.StickyFunc.lambda$new$0(StickyFunc.java:70)
	at org.cactoos.func.StickyBiFunc.apply(StickyBiFunc.java:93)
	at org.cactoos.func.StickyFunc.apply(StickyFunc.java:77)
	at org.cactoos.scalar.Sticky.value(Sticky.java:79)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.iterator.Shuffled.hasNext(Shuffled.java:77)
	at org.cactoos.iterator.Joined.hasNext(Joined.java:92)
	at java.base/java.lang.Iterable.forEach(Iterable.java:74)
	at org.cactoos.list.ListOf.<init>(ListOf.java:66)
	at org.cactoos.list.ListOf.<init>(ListOf.java:56)
	at org.cactoos.iterator.Shuffled.lambda$new$0(Shuffled.java:67)
	at org.cactoos.scalar.Sticky.lambda$new$0(Sticky.java:73)
	at org.cactoos.func.StickyFunc.lambda$new$0(StickyFunc.java:70)
	at org.cactoos.func.StickyBiFunc.apply(StickyBiFunc.java:93)
	at org.cactoos.func.StickyFunc.apply(StickyFunc.java:77)
	at org.cactoos.scalar.Sticky.value(Sticky.java:79)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.iterator.Shuffled.hasNext(Shuffled.java:77)
	at java.base/java.lang.Iterable.forEach(Iterable.java:74)
	at org.cactoos.list.ListOf.<init>(ListOf.java:66)
	at org.cactoos.iterable.Sticky.lambda$new$0(Sticky.java:58)
	at org.cactoos.scalar.Sticky.lambda$new$0(Sticky.java:73)
	at org.cactoos.func.StickyFunc.lambda$new$0(StickyFunc.java:70)
	at org.cactoos.func.StickyBiFunc.apply(StickyBiFunc.java:93)
	at org.cactoos.func.StickyFunc.apply(StickyFunc.java:77)
	at org.cactoos.scalar.Sticky.value(Sticky.java:79)
	at org.cactoos.scalar.Mapped.lambda$new$0(Mapped.java:58)
	at org.cactoos.scalar.ScalarEnvelope.value(ScalarEnvelope.java:53)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.iterable.IterableOf.iterator(IterableOf.java:83)
	at org.cactoos.iterable.IterableEnvelope.iterator(IterableEnvelope.java:53)
	at org.cactoos.iterable.Synced.iterator(Synced.java:88)
	at org.eolang.lints.Program.defects(Program.java:104)
	at org.eolang.maven.LintMojo.lint(LintMojo.java:278)
	at org.eolang.maven.LintMojo.lambda$lintOne$1(LintMojo.java:148)
	at org.cactoos.scalar.ScalarOf.lambda$new$1(ScalarOf.java:71)
	at org.cactoos.scalar.ScalarEnvelope.value(ScalarEnvelope.java:53)
	at org.cactoos.text.TextOfScalar.asString(TextOfScalar.java:58)
	at org.cactoos.text.TextEnvelope.asString(TextEnvelope.java:51)
	at org.cactoos.bytes.BytesOf.lambda$new$2(BytesOf.java:233)
	at org.cactoos.bytes.BytesOf.asBytes(BytesOf.java:391)
	at org.cactoos.io.InputOf.lambda$new$5(InputOf.java:288)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.io.InputOf.lambda$new$6(InputOf.java:289)
	at org.cactoos.io.InputOf.stream(InputOf.java:312)
	at org.cactoos.io.TeeInput.stream(TeeInput.java:867)
	at org.cactoos.scalar.LengthOf.lambda$new$2(LengthOf.java:90)
	at org.cactoos.scalar.ScalarEnvelope.value(ScalarEnvelope.java:53)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.eolang.maven.footprint.Saved.value(Saved.java:109)
	at org.eolang.maven.footprint.FpGenerated.apply(FpGenerated.java:51)
	at org.eolang.maven.footprint.FpUpdateBoth.lambda$new$0(FpUpdateBoth.java:50)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpFork.apply(FpFork.java:74)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpFork.apply(FpFork.java:74)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpFork.apply(FpFork.java:74)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpExistedSource.lambda$new$0(FpExistedSource.java:44)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.footprint.FpEnvelope.apply(FpEnvelope.java:50)
	at org.eolang.maven.LintMojo.lintOne(LintMojo.java:153)
	at org.eolang.maven.LintMojo.lambda$exec$0(LintMojo.java:99)
	at org.eolang.maven.util.Threaded.lambda$total$0(Threaded.java:75)
	at org.cactoos.scalar.CallableEnvelope.call(CallableEnvelope.java:53)
	at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
	at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
	at java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)
	at java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:113)
	at java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:270)
	at org.cactoos.experimental.Threads.lambda$new$1(Threads.java:93)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:56)
	at org.cactoos.scalar.Checked.value(Checked.java:75)
	at org.cactoos.scalar.IoChecked.value(IoChecked.java:63)
	at org.cactoos.scalar.Unchecked.value(Unchecked.java:56)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:57)
	at org.cactoos.experimental.Threads.lambda$new$4(Threads.java:197)
	at org.cactoos.iterable.IterableEnvelope.iterator(IterableEnvelope.java:53)
	... 92 more
```

This happens because `lints` `0.0.38` uses `xnav` `0.1.8`, while `eo-parent` declares `0.1.9`. Let's fix this.

--- a/pom.xml
+++ b/pom.xml
@@ -128,7 +128,7 @@ SOFTWARE.
       <dependency>
         <groupId>com.github.volodya-lombrozo</groupId>
         <artifactId>xnav</artifactId>
-        <version>0.1.9</version>
+        <version>0.1.8</version>
       </dependency>
       <dependency>
         <groupId>org.apache.commons</groupId>

